src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Restore/GetAzureRmRecoveryServicesBackupWorkloadRecoveryConfig.cs

```diff
@@ -300,7 +300,8 @@ public string GetResourceId()
                 resourceIdentifier.ResourceGroupName = RecoveryPoint.ContainerName.Split(new string[] { ";" }, StringSplitOptions.None)[1];
                 resourceIdentifier.ResourceType = "/VMAppContainer";
```

Comment:
```suggestion
                resourceIdentifier.ResourceType = "Microsoft.Compute/virtualMachines";
```
and still `return resourceIdentifier.ToString();` should fix your problem